src/cljs/main/broadfcui/config.cljs

```diff
@@ -40,8 +40,6 @@
 
 (defn api-url-root [] (get @config "apiUrlRoot"))
 (defn debug? [] (get @config "isDebug"))
-(defn when-debug [x]
```

Comment:
We probably don't have to get rid of this function. It might still be useful someday.

Reply:
As a rule, we try not to leave "it might come in handy someday" functions in our codebase.